ffcx/codegeneration/access.py

```diff
@@ -376,10 +388,10 @@ def facet_edge_vectors(self, mt, tabledata, num_points):
 
         # Get edge vertices
         facet = self.symbols.entity("facet", mt.restriction)
-        facet_edge = mt.component[0]
+        edge_ridge = mt.component[0]
```

Comment:
Should this variable have changed? Looks like it may be a Find & Replace gone rogue